plugins/syntax/g_numeral.mlg

```diff
@@ -34,23 +34,8 @@ VERNAC ARGUMENT EXTEND numnotoption
 END
 
 VERNAC COMMAND EXTEND NumeralNotation CLASSIFIED AS SIDEFF
-  | #[ locality = Attributes.locality; ] ![proof][ "Numeral" "Notation" reference(ty) reference(f) reference(g) ":"
+  | #[ locality = Attributes.locality; ] [ "Numeral" "Notation" reference(ty) reference(f) reference(g) ":"
       ident(sc) numnotoption(o) ] ->
 
-    { (* It is a bug to use the proof context here, but at the request of
-       * the reviewers we keep this broken behavior for now. The Global env
-       * should be used instead, and the `env, sigma` parameteter to the
-       * numeral notation command removed.
-       *)
-      fun ~pstate ->
-        let sigma, env = match pstate with
-        | None ->
-          let env = Global.env () in
-          let sigma = Evd.from_env env in
-          sigma, env
-        | Some pstate ->
-          Pfedit.get_current_context pstate
-        in
-      vernac_numeral_notation env sigma (Locality.make_module_locality locality) ty f g (Id.to_string sc) o;
-      pstate }
+    { vernac_numeral_notation (Locality.make_module_locality locality) ty f g (Id.to_string sc) o }
```

Comment:
Change of behavior cc: @maximedenes

Reply:
The old behaviour makes no sense, the only thing you can do with it is get an error like
~~~coq
Local Set Primitive Projections.
Record wuint := wrap { unwrap : Decimal.uint }.
Declare Scope wuint_scope.
Delimit Scope wuint_scope with wuint.

Lemma foo (wrap:nat) : True. Proof.
Numeral Notation wuint wrap unwrap : wuint_scope.
(* Error: wrap should go from Decimal.int to wuint or (option wuint). *)
~~~

Reply:
I agree; I fixed this in the past but I was forced to revert the fix.